lib/ansible/errors/__init__.py

```diff
@@ -252,11 +252,35 @@ def __init__(self, message="", obj=None, show_content=True, suppress_extended_er
                                                   suppress_extended_error=suppress_extended_error, orig_exc=orig_exc)
 
 
-class AnsibleActionSkip(AnsibleRuntimeError):
```

Comment:
We decided that we don't like using exceptions as flow control.  But we don't need to fix this now.  We will add comments here that say that other people shouldn't copy this pattern.  We are going to change the action plugins that are using this to use a context manager to manager their temp directory as a replacement for this in the near future (2.6, probably) and revert the definition of these exceptions.

Reply:
this is done.